src/ReduxStore/FeedDetails/fetchInitCommentData.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+
+
+const InternalfetchInitCommentData = (props) => {
```

Comment:
This doesn't have to be a component, it can be just a function, with parameter.

Reply:
fixed